v0.5.5 breaks compatibility with Java 8 and 11

Applying the plugin at v0.5.5 to a project built under a Java version older than 17, e.g., Java 11, causes a startup error:
```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project '2024-oss-community-component-scanning-slides'.
> Could not resolve all artifacts for configuration ':classpath'.
   > Could not resolve org.m2ci.msp:gradle-pandoc-reveal-plugin:0.5.5.
     Required by:
         root project : > org.m2ci.msp.pandocreveal:org.m2ci.msp.pandocreveal.gradle.plugin:0.5.5
      > Dependency requires at least JVM runtime version 17. This build uses a Java 11 JVM.

```
This did not occur with older releases of this plugin.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,6 +4,10 @@ Gradle Pandoc reveal.js plugin
 [Unreleased]
 ------------
 
+### Fixed
+
+- Compatibility with projects built under Java 8 or 11
+
 ### Changed
 
 - [all changes since v0.5.5]

--- a/build.gradle
+++ b/build.gradle
@@ -17,6 +17,12 @@ dependencies {
     implementation libs.snakeyaml
 }
 
+java {
+    toolchain {
+        languageVersion = JavaLanguageVersion.of(8)
+    }
+}
+
 gradlePlugin {
     website = project.url
     vcsUrl = project.url

--- a/settings.gradle
+++ b/settings.gradle
@@ -1,3 +1,7 @@
+plugins {
+    id 'org.gradle.toolchains.foojay-resolver-convention' version '0.8.0'
+}
+
 rootProject.name = 'gradle-pandoc-reveal-plugin'
 
 dependencyResolutionManagement {

